fa25-team-b/cleaners/race_norm_corrections.py

```
@@ -0,0 +1,133 @@
+    'native_american_alaskan_native',
+    'white',
+    'white_MENA',
+    'hispanic',
+    'choose_not_to_answer', # will be removed later
+    'unknown'
+]
+
+#%% Original Counts
+
+dummy_cols = [col for col in df.columns if col.startswith('dummy_')]
+dummy_cols_and_mixed_col = dummy_cols + ['is_mixed_race']
```

Ruff (0.14.3)

Consider `[*dummy_cols, 'is_mixed_race']` instead of concatenation

Replace with `[*dummy_cols, 'is_mixed_race']`

(RUF005)